SetupGuide_Plugin_Admin.php

```diff
@@ -189,7 +189,7 @@ public function test_pgcache() {
 	 * @see \W3TC\Config::get_string()
 	 */
 	public function get_pgcache_settings() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the phpcs comment to have it validate the change.

Reply:
Removed PHPCS disable comment.